docs/reference/overrides/slider-root.json

```diff
@@ -1,6 +1,9 @@
 {
```

Comment:
@vladmoroz The `*.Value` components also support children as a function (with some props) for formatting: https://github.com/mui/base-ui/blob/master/docs/data/components/select/select.mdx?plain=1#L179-L183

Should we add them to `docs/reference/overrides` and displayed as well?

Reply:
> Should we add them to docs/reference/overrides to show them in the props table as well?

Yes please!